server/index.js

```diff
@@ -2,7 +2,7 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var db = require('../database');
 var app = express();
-app.use(express.static(__dirname + '/public'));
+app.use(express.static(__dirname + '/../public'));
 app.use(bodyParser.json());
 var port = 8888;
```

Comment:
This might only be needed when we setup our proxy server, but you can do `port = process.env.PORT || 8888` to ensure you're using the right port on deployment.

Reply:
Gotcha, I'll look into that